com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md

```diff
@@ -148,6 +148,8 @@ The **Lighting** panel has tools that you can use to visualize various component
 | **Fullscreen Debug Mode**            | Use the drop-down to select a fullscreen lighting effect to debug. For example, you can visualize [Contact Shadows](Override-Contact-Shadows.md), the depth pyramid, and indirect diffuse lighting. |
 | **Tile/Cluster Debug**               | Use the drop-down to select an internal HDRP lighting structure to visualize on screen.<br/>&#8226; **None**: Select this option to turn off this debug feature.<br/>&#8226; **Tile**: Select this option to show an overlay of each lighting tile, and the number of lights in them.<br/>&#8226; **Cluster**: Select this option to show an overlay of each lighting cluster that intersects opaque geometry, and the number of lights in them.<br/>&#8226; **Material Feature Variants**: Select this option to show the index of the lighting Shader variant that HDRP uses for a tile. You can find variant descriptions in the *lit.hlsl* file. |
 | **- Tile/Cluster Debug By Category** | Use the drop-down to select the Light type that you want to show the Tile/Cluster debug information for. The options include [Light Types](Light-Component.md), [Decals](Decal-Projector.md), and [Density Volumes](Density-Volume.md).<br/>This property only appears when you select **Tile** or **Cluster** from the **Tile/Cluster Debug** drop-down. |
+| **- Cluster Debug Mode** | Use the drop-down to select the visualization mode for the cluster. You can choose either **VisualizeOpaque** which will show cluster information on opaque geometry or **VisualizeSlice** which will show cluster information at a set distance from the camera. This property only appears when you select **Cluster** from the **Tile/Cluster Debug** drop-down. |
+| **- Cluster Distance** | Use this slider to choose at which distance from the camera  the cluster slice should be displayed. This property only appears when you select **VisualizeSlice** from the **Cluster Debug Mode** drop-down. |
```

Comment:
Use this slider to set the distance from the camera at which to display the cluster slice.<br/>This property only appears when you select \*\*VisualizeSlice** from the \*\*Cluster Debug Mode** drop-down.